soundfile.py

```diff
@@ -986,7 +986,9 @@ def _open(self, file, mode_int, closefd):
             file_ptr = _snd.sf_open(file, mode_int, self._info)
         elif isinstance(file, int):
             file_ptr = _snd.sf_open_fd(file, mode_int, self._info, closefd)
-        elif all(hasattr(file, a) for a in ('seek', 'read', 'write', 'tell')):
+        elif ( hasattr(file, 'seek') and hasattr(file, 'tell') and
```

Comment:
This improves readability as it helps differentiate the inner parens from the outer parens. I will not take pep8 as gospel.